conf/i18n/translatehelpervisitor.js

```diff
@@ -0,0 +1,163 @@
+const Handlebars = require('handlebars');
+const TranslationPlaceholder = require('./models/translationplaceholder');
+
+/**
+ * TranslateHelperVisitor accepts a handlebars AST, and replaces all translate placeholders
+ * with either static translations, or a runtime translation helper.
+ */
+class TranslateHelperVisitor {
```

Comment:
Can this class extend Handlebars.Visitor? Or will that not work

Reply:
I wrote it that way originally but then I remembered our hopy mantra of composition over inheritance. I was pretty split on which way was better, so if you prefer extending here I’ll swap it!

Reply:
Gotcha. Would extending that class allow us to remove some boilerplate or not really change the class much? Yeah, we do usually favor composition over inheritance. If extending the class wouldn't really get us anything in terms of removing boilerplate, than yeah, let's compose.

Reply:
it lets use remove the accept() but not much else is changed besides that

Reply:
Gotcha, let's leave as is then.